src/addons/plugin/property-proxy.js

```diff
@@ -15,18 +15,16 @@ const Property = require('../property');
 
 class PropertyProxy extends Property {
   constructor(device, propertyName, propertyDict) {
-    super(device, propertyName, propertyDict.type);
-
-    if (propertyDict.description) {
-      this.description = propertyDict.description;
-    }
-    if (propertyDict.unit) {
-      this.description = propertyDict.unit;
-    }
+    super(device, propertyName, propertyDict);
 
     this.value = propertyDict.value;
 
     this.propertyChangedPromises = [];
+    this.propertyDict = {};
+  }
+
+  asDict() {
+    return Object.assign({}, this.propertyDict, super.asDict());
```

Comment:
This is going to overlay the base class properties on top of this object's properties. Was that intended?

Reply:
I don't think it does what you said. Is overlays, the contents of this.propertyDict with the contents of the base classes asDict method, neither of which includes this object's properties.

The same applies to your previous comment about deviceDict.